Core/src/org/sleuthkit/autopsy/modules/hashdatabase/HashDbManager.java

```diff
@@ -786,48 +1031,340 @@ public boolean equals(Object obj) {
+        void setSendIngestMessages(boolean showInboxMessages) {
+            this.sendIngestMessages = showInboxMessages;
+        }
+
+        /**
+         * Indicates whether the hash database accepts updates.
+         *
+         * @return True if the database accepts updates, false otherwise.
+         *
+         * @throws org.sleuthkit.datamodel.TskCoreException
+         */
+        @Override
+        public boolean isUpdateable() throws TskCoreException {
+            return false;
+            // TEMP this will change as soon as adding to the database is supported 
+            // return (! readOnly);
+        }
+
+        /**
+         * Adds hashes of content (if calculated) to the hash database.
+         *
+         * @param content The content for which the calculated hashes, if any,
+         *                are to be added to the hash database.
+         *
+         * @throws TskCoreException
+         */
+        @Override
+        public void addHashes(Content content) throws TskCoreException {
+            addHashes(content, null);
+        }
+
+        /**
+         * Adds hashes of content (if calculated) to the hash database.
+         *
+         * @param content The content for which the calculated hashes, if any,
+         *                are to be added to the hash database.
+         * @param comment A comment to associate with the hashes, e.g., the name
+         *                of the case in which the content was encountered.
+         *
+         * @throws TskCoreException
+         */
+        @Override
+        public void addHashes(Content content, String comment) throws TskCoreException {
+            
+        }
+
+        /**
+         * Adds a list of hashes to the hash database at once
+         *
+         * @param hashes List of hashes
+         *
+         * @throws TskCoreException
+         */
+        @Override
+        public void addHashes(List<HashEntry> hashes) throws TskCoreException {
+            
+        }
+
+        /**
+         * Perform a basic boolean lookup of the file's hash.
+         *
+         * @param content
+         *
+         * @return True if file's MD5 is in the hash database
+         *
+         * @throws TskCoreException
+         */
+        @Override
+        public boolean lookupMD5Quick(Content content) throws TskCoreException {
+            // This only works for AbstractFiles and MD5 hashes 
+            assert content instanceof AbstractFile;
+            if (content instanceof AbstractFile) {
+                AbstractFile file = (AbstractFile) content;
+                if (null != file.getMd5Hash()) {
+                    try{
+                        return EamDb.getInstance().isFileHashInReferenceSet(file.getMd5Hash(), this.referenceSetID);
+                    } catch (EamDbException ex){
+                        Logger.getLogger(SleuthkitHashSet.class.getName()).log(Level.SEVERE, "Error performing central reposiotry hash lookup", ex); //NON-NLS
```

Comment:
I think it could help in debugging if we identify the database that experienced the error.